lib/fedora_migrate/repository_migrator.rb

```diff
@@ -52,6 +37,32 @@ def get_source_objects
 
     private
 
+    def migrate_object source
```

Comment:
Yes, the mover should true true, or raise an error which will get collected by the migrator and `@failed` is incremented.